pkg/fqdn/dns/dns.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dns
+
+import "strings"
+
+// These functions were copied and adapted from github.com/miekg/dns.
```

Comment:
I think you need to [include a copyright notice](https://github.com/miekg/dns/blob/master/LICENSE#L7).

Reply:
Thanks, good point. I've adjusted the copyright comment in that file accordingly.